### PR TITLE
block meshing

### PR DESCRIPTION
**Pull request by LinusKnecht**
## Related issues
from PR #20.

## Description
This PR extends ```gustaf.create``` by  ```faces```  and ```volumes```. Simple routines to create a 2D-structured quadrangle block mesh mesh and a 3D-structured hexahedron block mesh. Functions are:

```
    faces.quad_block_mesh(bounds,resolutions)
    volumes.hexa_block_mesh(bounds, resolutions)
```
A small example ```create_block_mesh.py``` is provided.

### CODE REVIEW

File: gustaf/create/faces.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    Returns
+    --------
+    face_mesh: Volumes
+    """
+    assert np.array(bounds).shape == (2, 2), \
+        "bounds array must have 2x2 entries."
+    assert len(resolutions) == 2, \
+        "resolutions array must have two entries."
+    assert np.greater(resolutions, 1).all(), \
+        "All resolutions must be at least 2."
```
Comment (clemens-fricke) [inline gustaf/create/faces.py:29]:
```suggestion
    if np.array(bounds).shape != (2, 2):
        raise ValueException("Bounds must have a dimension of (2,2).")
    if len(resolutions) != 2:
        raise ValueException("Resolutions  must have two entries.")
    if not np.greater(resolutions, 1).all():
        raise ValueException("All resolutions values must be at least 2.")
```
assert does not conform to the packages contributing guidelines. Please rewrite in above style.

File: gustaf/create/volumes.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    Returns
+    --------
+    volume_mesh: Volumes
+    """
+    assert np.array(bounds).shape == (2, 3), \
+        "bounds array must have 2x3 entries."
+    assert len(resolutions) == 3, \
+        "resolutions array must have three entries."
+    assert np.greater(resolutions, 1).all(), \
+        "All resolutions must be at least 2."
```
Comment (clemens-fricke) [inline gustaf/create/volumes.py:34]: Pls, use exceptions instead of assert.

**Review by clemens-fricke — COMMENTED** [review]
Nice. Thank you for also providing an example.

**Review by j042 — COMMENTED** [review]
Thank you!

**Review by LinusKnecht — COMMENTED** [review]
thank you!

**Review by j042 — APPROVED** [review]
Great

**Pull request merged**